pom.xml

```diff
@@ -8,7 +8,7 @@
         <version>7</version>
     </parent>
 
-    <packaging>jar</packaging>
+    <packaging>bundle</packaging>
```

Comment:
What impact does this have on how the library is consumed via Maven Central for non-OSGI users?